editor-components/Image_gallery/image_gallery_5/image-gallery5.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from "react";
+import { BaseImageGallery } from "../../EditorComponent";
+import styles from "./image-gallery5.module.scss";
+
+interface ImageType {
+  type: string;
+  key: string;
+  displayer: string;
+  value: string;
+}
+
+class ImageGalleryComponent5 extends BaseImageGallery {
+  constructor(probs?: any) {
+    super(probs, styles);
+
+    this.addProp({
+      type: "array",
+      key: "gallery",
+      displayer: "Gallery",
+      value: [
+        {
+          type: "image",
+          key: "imageGallery",
+          displayer: "Image1",
+          value: "https://arty.flatheme.net/assets/images/p-clean-2-1.jpg"
```

Comment:
Change all this image URLs with spica storage URL.

You can get these URLs with uploading the images.